main.py

```diff
@@ -515,72 +558,34 @@ def main(cl_arguments):
                 batch_size=args.batch_size,
                 weighting_method=args.weighting_method,
                 scaling_method=args.scaling_method,
-                train_params=to_train,
+                train_params=pre_target_train_model_state,
```

Comment:
This doesn't seem right. Isn't this supposed to be a list of parameters, not a model state?

Reply:
Might just be a naming issue.

Reply:
Yep, it's a list of parameters - will change the name.